src/org/exoplatform/ui/login/tasks/LoginTask.java

```diff
@@ -75,6 +75,12 @@ public Integer doInBackground(String... params) {
   }
 
   @Override
+  protected void onCancelled() {
```

Comment:
onCanceled(Result result) only available from API 11, we will need check it later after increased the minimum API